stores/shoppingCart.ts

```diff
@@ -16,27 +17,18 @@ export const useShoppingCartStore = defineStore('shoppingCart', {
     items: items(),
     itemToBuy: undefined,
   }),
-  getters: {
-    getItems: () => items(),
-    getItemsByPrefix: () => (prefix: string) =>
-      items().filter((item) => item.urlPrefix === prefix),
-    getItem: () => (id: ID) => items().find((item) => item.id === id),
-    isItemInCart: () => (id: ID) =>
-      items().find((item) => item.id === id) !== undefined,
-    getItemToBuy: ({ itemToBuy }) => itemToBuy,
-  },
-
   actions: {
+    getItemsByPrefix(prefix: string) {
+      return this.items.filter((item) => item.urlPrefix === prefix)
+    },
+    isItemInCart(id: ID) {
+      return this.items.find((item) => item.id === id) !== undefined
+    },
```

Comment:
all these are getters, why did you move them to actions?

Reply:
It's just to be more consistent with pinia's documentations. Usually getters are used similar to computedRefs, but here we're accepting parameters.